packages/stacked_cli/lib/src/services/process_service.dart

```diff
@@ -27,10 +27,26 @@ class ProcessService {
   ///
   /// Args:
   ///   appName (String): The name of the app that's going to be create.
-  Future<void> runCreateApp({required String appName}) async {
+  Future<void> runCreateApp(
+      {required String organization,
+      required String platforms,
+      required String appName}) async {
+    List<String> arguments = [];
+    arguments.add(ksCreate);
+
+    if (organization.isNotEmpty) {
+      arguments.addAll([ksOrganization, organization]);
+    }
+
+    if (platforms.isNotEmpty) {
+      arguments.addAll([ksPlatform, platforms]);
+    }
```

Comment:
This pattern is causing a lot of additional code that's not required. you can simply make the value null and use null to know if a value was added or not. Making it required and passing empty is not a good pattern.